Chapter-4-applying-tactical-domain-driven-design/Fitnet.Common/Fitnet.DomainDrivenDesign.BuildingBlocks/IDomainEvent.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.DomainDrivenDesign.BuildingBlocks;
+
+/// <summary>
+/// Represents a domain event, which encapsulates a significant occurrence or state change within the domain model.
+/// </summary>
+public interface IDomainEvent
+{
+    /// <summary>
+    /// Gets the unique identifier of the domain event.
+    /// </summary>
+    Guid Id { get; }
+
+    /// <summary>
+    /// Gets the timestamp when the domain event occurred.
+    /// </summary>
+    DateTime OccurredOn { get; }
```

Comment:
Change to OccuredAt (better from grammar perspective)

Reply:
Nice. I will change it.